features/step_definitions/journey_test_steps.rb

```diff
@@ -877,10 +877,10 @@ def filter_to_check(filter_name,filter_value,filter_exist)
 end
 
 Then /I am on a page with '(.*)' in search summary text$/ do |value|
+  find(:xpath, "//*[@class='search-summary']/em[1]").text().should == normalize_whitespace(value)
```

Comment:
Might be a good idea to leave a comment here because the implication of the change is non-obvious?